quickwit-swim/src/state.rs

```diff
@@ -329,10 +329,15 @@ impl ArtilleryEpidemic {
         use Request::*;
 
         if message.cluster_key == self.config.cluster_key {
-            // We want to abort if a new member has the same node_id of an existing member.  
+            // We want to abort if a new member has the same node_id of an existing member.
```

Comment:
I'm wondering why there is a change in this file. Is this coming from `make fmt`?

Reply:
probably because of the nighlty fmt